command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -8,13 +9,27 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
+        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
 
     def raised(self, alarm):
         self._killed = True
+        self.bagger_dump()
 
     def cleared(self, alarm):
         self._killed = False
 
+    def bagger_dump(self):
+        """Call online_bagger/dump service"""
+        camera = '/camera/front/left/camera_info /camera/front/left/image_raw '
+        navigation = '/wrench /wrench_actual /c3_trajectory_generator/trajectory_v /c3_trajectory_generator/waypoint /trajectory '
+        controller = '/pd_out /rise_6dof/parameter_descriptions /rise_6dof/parameter_updates '
+        blueview = '/blueview_driver/ranges /blueview_driver/image_color'
+        kill_topics = camera + navigation + controller + blueview
+        try:
+            bag_status = self.bagging_server(bag_name='kill_bag', bag_time=60, topics=kill_topics)
```

Comment:
It didn't work when I tried to run them simultaneously, if it was in the middle of a service call, it seems that it would start running the second one in the middle of the first one and the resulting two bags would have the topics split between the two service calls (i.e. I had one bag that had one of the camera topics, and another than had the other half, both had the same amount of messages).

I fixed the issue with a PR to mil_common, if the service is already being called it will reject the all and give priority to the current one.